src/main/java/medicaldoctor/Program.java

```diff
@@ -10,11 +10,12 @@
 import medicaldoctor.core.DatabaseScope;
 
 public class Program extends Application {
+    
     @Override
     public void start(Stage stage) throws Exception {
         ControllerManager cm = new ControllerManager();
-        cm.load(LookUp.LOGIN_SCREEN_NAME, LookUp.LOGIN_SCREEN);
-        cm.showScreen(LookUp.LOGIN_SCREEN_NAME);
+        cm.load(LookUp.LOGIN_SCREEN);
```

Comment:
could this and the next line be collapsed into one method?

Reply:
No, because there are cases where you won't showscreen directly after a load. For example, when you want to go back to something you will just call showScreen(key)

Reply:
ok